vnext/Microsoft.ReactNative.Cxx.UnitTests/JSValueTest.cpp

```diff
@@ -4,13 +4,13 @@
 #include "pch.h"
 #include "JSValue.h"
 #include "JsonJSValueReader.h"
-#include "catch.hpp"
 
 namespace winrt::Microsoft::ReactNative {
 
-TEST_CASE("TestReadObject", "JSValueTest") {
-  const wchar_t *json =
-      LR"JSON({
+TEST_CLASS (JSValueTest) {
```

Comment:
>JSValueTest [](start = 12, length = 11)

I don't see these tests running on the CI loops. This is likely related to the same issue described in [https://github.com/microsoft/react-native-windows/issues/3552](https://github.com/microsoft/react-native-windows/issues/3552).

Reply:
Also I've had calling convention mismatches trying to get Google Test to run with our x86 build ([https://github.com/microsoft/react-native-windows/issues/2702](https://github.com/microsoft/react-native-windows/issues/2702)). We'll need to resolve these issues in the future.

---
In reply to: [375000402](https://github.com/microsoft/react-native-windows/pull/4030#discussion_r375000402) [](ancestors = 375000402)

Reply:
I would like them to start running. We obviously do not support the Catch2, but I expected that we already run GTest. It is odd to learn that we do not run GTest either. What type of unit tests we are running if any?

Reply:
If I remember correctly, we override the installed Visual Studio Test Platform with the latest version because we need it for integration tests. This latest test platform, however, does not work properly with Google Test. (@JunielKatarn could you please confirm this?) One possible solution would be to install the latest test platform after we finish running the unit tests and before we ran the integration tests.